### ISSUE TITLE
No possibility to delete "around" or "inside" a text object

### ISSUE THREAD

**Issue by Sawiq** [egaga/openvim#77]
There is no way to delete with an `a` or `i` modifier like so:
* `da(` should delete everything inside parenthesis with surrounding whitespace
* `di"` should delete everything inside current quotation marks.

and so on

**Comment by egaga** [comment]
True, those are useful commands. However, probably won't be much expanding Openvim myself.
Thanks for the suggestion, anyway.

**Comment by Sawiq** [comment]
No problem, VIM is very complicated and I understand, that it is almost impossible to reproduce all it's functionality. I regret, that I am unable to help with implementation myself. Maybe someone will pick up from here in the future :)

Very interesting piece of software which and a nice introduction for new users.

**Comment by egaga** [comment]
Thank you, hopefully someone could, albeit the implementation has its quirks as this was put together as more of a demonstration, but it's not a lost cause. And maybe it's enough for what it's main purpose is: to get one's feet wet for a dry run :)
